index.bs

```diff
@@ -1649,6 +1721,9 @@ interface MLGraphBuilder {
   // Create a scalar operand from the specified number of the specified type.
   MLOperand constant(MLOperandDataType type, MLNumber value);
 
+  // Create an operand from a specified tensor.
+  MLOperand constant(MLConstantTensor tensor);
```

Comment:
Ah, so I see one place where having a named `MLConstantTensor` specifically makes the function parameter requirement clearer. (resolve me)